<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QSlider" name="horizontalSlider">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
        <property name="value">
         <number>9</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>60</y>
          <width>160</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>StartTime</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>EndTime</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Ignore Weekends</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOnderzoek_Delft">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_File"/>
    <addaction name="actionSafe_data"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuSet_Column_To_Type">
     <property name="title">
      <string>Set Column To Type</string>
     </property>
     <addaction name="actionCO2_Sensor"/>
     <addaction name="actionPIR_Sensor"/>
     <addaction name="actionAirflow"/>
     <addaction name="actionLightstate"/>
     <addaction name="actionTemperature"/>
    </widget>
    <addaction name="menuSet_Column_To_Type"/>
    <addaction name="actionFill_Empty_Cells"/>
   </widget>
   <addaction name="menuOnderzoek_Delft"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionAdd_File">
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSafe_data">
   <property name="text">
    <string>Safe data</string>
   </property>
  </action>
  <action name="actionCO2_Sensor">
   <property name="text">
    <string>CO2 Sensor</string>
   </property>
  </action>
  <action name="actionPIR_Sensor">
   <property name="text">
    <string>PIR Sensor</string>
   </property>
  </action>
  <action name="actionAirflow">
   <property name="text">
    <string>Airflow</string>
   </property>
  </action>
  <action name="actionLightstate">
   <property name="text">
    <string>Lightstate</string>
   </property>
  </action>
  <action name="actionTemperature">
   <property name="text">
    <string>Temperature</string>
   </property>
  </action>
  <action name="actionFill_Empty_Cells">
   <property name="text">
    <string>Fill Empty Cells</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
